src/main/kotlin/no/nav/syfo/melding/MeldingService.kt

```diff
@@ -70,14 +70,15 @@ class MeldingService(
             PdfContent(it.pdf)
         }
 
-    private fun getBehandlerRefForConversation(conversationRef: UUID, personIdent: PersonIdent): UUID {
+    private fun getBehandlerRefForConversation(meldingFraBehandler: MeldingFraBehandler, personIdent: PersonIdent): UUID {
         return getUtgaendeMeldingerInConversation(
-            conversationRef = conversationRef,
+            conversationRef = meldingFraBehandler.conversationRef,
             personIdent = personIdent,
         )
             .firstOrNull()
             ?.behandlerRef
-            ?: throw IllegalStateException("Fant ikke behandlerRef for samtale $conversationRef, kunne ikke knyttes til melding fra behandler")
+            ?: meldingFraBehandler.behandlerRef
```

Comment:
Vil ikke denne alltid være null? 
Tenker vi ikke vil kalle getBehandlerRefForConversation for innkommende meldinger av type `HENVENDELSE_MELDING_TIL_NAV` og så gjøre `behandlerRef` over her nullable

Reply:
Jo, dette var et eksperiment der jeg hadde satt behandlerRef på en rad i databasen for å se hvordan det virker. Men enig i at vi i første omgang kan forsøke å la denne være null.